README.md

```diff
@@ -35,6 +35,26 @@ to your project or organization:
   command, because [changing default branch of a repository currently requires administrative token rights](https://developer.github.com/v3/repos/#update-a-repository).
   You can generate a token from your [personal access tokens page](https://github.com/settings/tokens/new).
 
+> ### Triggering release workflow events
```

Comment:
Is the quoting intentional?

Reply:
Wanted it to stand out. I can remove if you want.

Reply:
Really looks like an edge case - semantically, quoting means something very different than "standing out" :D